imblearn/utils/estimator_checks.py

```diff
@@ -283,6 +284,20 @@ def check_samplers_multiclass_ova(name, Sampler):
     assert_allclose(y_res, y_res_ova.argmax(axis=1))
 
 
+def check_samplers_2d_target(name, Sampler):
+    X, y = make_classification(
+        n_samples=100,
+        n_classes=3,
+        n_informative=4,
+        weights=[0.2, 0.3, 0.5],
+        random_state=0,
+    )
+
+    y = y.reshape(-1, 1)  # Make the target 2d
```

Comment:
Could you add another check when having pandas Series and DataFrame

Reply:
I was thinking to make a follow up PR to address #666. So, this check could be added in that same PR. What do you think?

Reply:
OK good then